packages/server/src/modules/user/modules/session/auth/crypto.js

```diff
@@ -2,7 +2,7 @@ import crypto from 'crypto';
 import settings from '../../../../../../../../settings';
 
 // Use password-based key derivation function to derive MAC key and encryption key from secret passphrase
-const _deriveSymmetricKey = salt => crypto.pbkdf2Sync(settings.user.secret, salt, 10000, 32, 'sha256');
+const _deriveSymmetricKey = salt => crypto.pbkdf2Sync(settings.user.secret || '', salt, 10000, 32, 'sha256');
```

Comment:
Why is this change here? We derive secret keys from nothing, this is a security hole?